packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -386,6 +386,57 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold_center}
+///
+/// Here, we will use a [Shortcuts] and [Actions] widget to add and remove from a counter.
+/// This can be done by creating a child widget that is focused and pressing the logical key
+/// sets that have been defined in [Shortcuts] and defining the actions that each key set
+/// performs.
+///
+/// ```dart imports
+/// import 'package:flutter/services.dart';
+/// ```
+///
+/// ```dart preamble
+/// class Increment extends Intent {}
+///
+/// class Decrement extends Intent {}
+/// ```
+///
+/// ```dart
+/// int count = 0;
+///
+/// Widget build(BuildContext context) {
+///   return Shortcuts(
+///     shortcuts: <LogicalKeySet, Intent> {
+///       LogicalKeySet(LogicalKeyboardKey.shift, LogicalKeyboardKey.numpadAdd): Increment(),
+///       LogicalKeySet(LogicalKeyboardKey.shift, LogicalKeyboardKey.numpadSubtract): Decrement(),
```

Comment:
Just using shift + numpadAdd/Subtract is a little tough for some laptop users (for example, Macbooks do not have a numpad). Maybe include a set or two that is more accessible? Doesn't have to be + or -, even a letter on the keyboard will do.

Reply:
How about `shift` and `keyK` for Increment and `shift` and `keyL` for Decrement?

Reply:
That should be good!